main.js

```diff
@@ -0,0 +1,106 @@
+let apiKey = 'YOUR_API_KEY';
+let ably = new Ably.Realtime({
+  key: apiKey,
+});
+
+let regularChannelName = "channel-" + Math.random().toString(36).substr(2, 16);
+let channelOpts = { params: { occupancy: "metrics" } };
+let channel = ably.channels.get(regularChannelName, channelOpts);
+let resultArea = document.getElementById("result");
+resultArea.scrollTop = resultArea.scrollHeight;
+
+channel.subscribe("[meta]occupancy", (msg) => {
+  let msgJSONobj = JSON.parse(JSON.stringify(msg));
+  console.log("MSG: " + msg);
+  //extract occupancy data from the message returned in the callback
+  let occupancyMetrics = msgJSONobj.data.metrics;
+  if (occupancyMetrics && msgJSONobj.name.includes("[meta]")) {
+    resultArea.value +=
```

Comment:
This may be cleaner to write as a template string instead

Reply:
Done.